core/node/node_framework/src/implementations/layers/query_eth_client.rs

```diff
@@ -72,16 +69,6 @@ impl WiringLayer for QueryEthClientLayer {
             } else {
                 None
             },
```

Comment:
why did you get rid of this? We need to watch the GW for EthWatch component. Or did you add it back somewhere else?

Reply:
ah its in query_gateway_client I think